src/components/ImageCard/_image-card.scss

```diff
@@ -40,8 +40,18 @@
 
 .cc-image-card__image {
   @extend %img-responsive;
-
+  @extend %img-cover;
```

Comment:
I've stopped short of refactoring this and creating a specific extend as it may well be rendered unnecessary by the following ticket which would ensure placeholders are always the same size as images they may be butting up against in lists
https://github.com/wellcometrust/corporate/issues/8063

Reply:
Thanks for the context Chris, very helpful